CMakeLists.txt

```diff
@@ -50,6 +50,8 @@ pybind11_add_module(idaklu
   pybamm/solvers/c_solvers/idaklu/CasadiSolverOpenMP_solvers.hpp
   pybamm/solvers/c_solvers/idaklu/casadi_sundials_functions.cpp
   pybamm/solvers/c_solvers/idaklu/casadi_sundials_functions.hpp
+  pybamm/solvers/c_solvers/idaklu/idaklu_jax.cpp
+  pybamm/solvers/c_solvers/idaklu/idaklu_jax.hpp
```

Comment:
Usually we add these files here for CMake to ask `pybind11` to create bindings for, but the corresponding list of `sources` in the `Extension` in `setup.py` does not get updated in accordance with these files. Could you add these there as well? We should probably add a comment in both places so that they go hand-in-hand and are in sync on further updates.